superset/db_engine_specs/base.py

```diff
@@ -1343,6 +1343,21 @@ def cancel_query(cls, cursor: Any, query: Query, cancel_query_id: str) -> bool:
 
         return False
 
+    @classmethod
+    def _has_view(
+        cls,
+        conn: Connection,
+        dialect: Dialect,
+        view_name: str,
+        schema: Optional[str] = None,
+    ):
+        view_names = dialect.get_view_names(connection=conn, schema=schema)
+        return view_name in view_names
+
+    @classmethod
+    def has_view(cls, engine: Engine, view_name: str, schema: Optional[str] = None):
```

Comment:
Could we move `has_view` to `superset.models.core` in order to follow `has_table` pattern.